examples/neural_compressor/optical-character-recognition/README.md

```diff
@@ -0,0 +1,49 @@
+<!---
+Copyright 2020 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Optical Character Recognition
+
+The script [`run_trocr.py`](https://github.com/huggingface/optimum-intel/blob/main/examples/neural_compressor/optical-character-recognition/run_trocr.py)
```

Comment:
I would rename the script to run_ocr.py in order to match what we currently have in the examples. And in the case we currently can only use TrOCR models I would specify it in the README

Reply:
Done